docs/book/faq.md

```diff
@@ -0,0 +1,13 @@
+# FAQ
+
+- I got the following error on installation, how to fix it?
```

Comment:
In a technical documentation there is no _higher_ "I". For a blog this may work, but not in a documentation.

Reply:
Do you have better wording suggestion?